appeals/web/src/server/appeals/appeal-details/start-case/__tests__/start-case.test.js

```diff
@@ -0,0 +1,35 @@
+import { parseHtml } from '@pins/platform';
+import nock from 'nock';
+import supertest from 'supertest';
+import { createTestEnvironment } from '#testing/index.js';
+import { appealData } from '#testing/appeals/appeals.js';
+
+const { app, teardown } = createTestEnvironment();
+const request = supertest(app);
+const baseUrl = '/appeals-service/appeal-details';
+
+describe('start-case', () => {
```

Comment:
Ideally there should be unit tests for the `POST` route as well as the `GET`

Reply:
Yes, but in this case user is not posting any data. The data is being posted is auto-generated "today" date

Reply:
On second look, there is actually a `POST` test - `should redirect to the confirmation page` - ideally that should be in its own `describe` but it's very minor